### ISSUE TITLE
Personal Wiki page

### ISSUE THREAD

**Issue by XWolfOverride** [takezoe/gitbucket#703]
I think i'ts a good idea to have a Wiki page per user (not only per repo). To allow users to make personal public or private pages.

**Comment by aadrian** [comment]
-1

GitBucket should not be transformed into a CMS

**Comment by mgeorge** [comment]
I agree with aadrian.  Users can already create their own repository 
just for their Wiki if needed.

**Comment by hansgru** [comment]
-1. Users can now make any private repos as they want and use them only for wiki if they wish.

Related:
GitHub has however "sites", but they're only public and are something different.

The way GitHub implemented the "sites" however is not quite user friendly (gh-pages branch). 

IMO GitBucket could do it much better, using the same convention as for it's wiki: 
```
repo
repo_WIKI
repo_SITE
```

**Comment by pibi** [comment]
@hansgru great idea! +1
